libs/application/templates/parental-leave/src/lib/answerValidators.ts

```diff
@@ -18,55 +22,78 @@ function buildValidationError(
 
 const FIRST_PERIOD_START = 'periods[0].startDate'
 const FIRST_PERIOD_END = 'periods[0].endDate'
+const FIRST_PERIOD_RATIO = 'periods[0].ratio'
 
+/**
+ * TODO
+ * Maybe it is enough to have a single 'periods' answer validator to handle all answers for periods?
+ * loop through all periods?
+ */
```

Comment:
this is definitely something I wondered about

Reply:
I haven't figured it all yet, I'm still going through everything to understand how everything is working together